components/projectsView/ProjectFilters.js

```diff
@@ -136,7 +136,7 @@ export default function ProjectFilters({
                 )
               }
             >
-              <option value="">Any workgroup</option>
+              <option value="">By workgroup</option>
```

Comment:
I also think it is a bit confusing to filter back to all workgroups by choosing "By workgroup", but I like it as a placeholder that states what the dropdown does. Maybe an extra option with the "All workgroups" wording that Chia mentioned?

Reply:
+1